### ISSUE TITLE
quickcheck-classes-0.4.14 breaks primitive's CI

### ISSUE THREAD

**Issue by RyanGlScott** [haskell/primitive#190]
From the [most recent build](https://travis-ci.org/haskell/primitive/jobs/407341878#L1424):

```
[31 of 31] Compiling Test.QuickCheck.Classes ( src/Test/QuickCheck/Classes.hs, /tmp/dist-test.NFki/dist-newstyle/build/x86_64-linux/ghc-8.4.3/quickcheck-classes-0.4.14/build/Test/QuickCheck/Classes.o )

src/Test/QuickCheck/Classes.hs:64:5: error:
    Not in scope: ‘semigroupoidLaws’
    Perhaps you meant ‘semigroupLaws’ (imported from Test.QuickCheck.Classes.Semigroup)
   |
64 |   , semigroupoidLaws
   |     ^^^^^^^^^^^^^^^^

src/Test/QuickCheck/Classes.hs:65:5: error:
    Not in scope: ‘commutativeSemigroupoidLaws’
    Perhaps you meant ‘commutativeSemigroupLaws’ (imported from Test.QuickCheck.Classes.Semigroup)
   |
65 |   , commutativeSemigroupoidLaws
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal: Failed to build quickcheck-classes-0.4.14 (which is required by
test:test from primitive-tests-0.1).
```

@andrewthad, do you know what is going on here?

**Comment by andrewthad** [comment]
Yes.

**Comment by andrewthad** [comment]
This is fixed in `quickcheck-classes-0.4.14.1`, which I just released to hackage.

**Comment by RyanGlScott** [comment]
Wonderful, things are [building again](https://travis-ci.org/haskell/primitive/builds/407341877).
